### DISCUSSION TITLE
How do you create hi quality figures in R for publication?

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1409] [Q&A]
We are trying to figure out how to increase quality of figures for Grace crab paper.

"We prefer replacement figures containing vector/editable objects rather than embed images. Preferred formats are eps,ai,and pdf. Hope this is possible."

see https://github.com/RobertsLab/resources/issues/1404

**Comment by kubu4** [comment]
Do you happen to have any more available details on this? I know they say "All figures are blurred.", but they don't look blurry to me (not that that matters).

Does the journal offer any guidelines on figure requirements (e.g. size, DPI, how to generate/manipulate, etc)?

How were the figures supplied in the first place? Looks like we had/have PDFs - is this not how the figures were provided?

**Reply by sr320** [reply]
All we got was here https://github.com/RobertsLab/resources/issues/1404#issue-1145252743

**Comment by kubu4** [comment]
How were the figures originally provided?

**Reply by sr320** [reply]
copy  and paste into a word doc.

**Comment by kubu4** [comment]
Ah! So those PDFs should be good then, right?
